lightly/active_learning/scorers/classification.py

```diff
@@ -93,10 +94,24 @@ class ScorerClassification(Scorer):
 
     """
     def __init__(self, model_output: Union[np.ndarray, List[List[float]]]):
-        if isinstance(model_output, List):
+        if not isinstance(model_output, np.ndarray):
             model_output = np.array(model_output)
+
+        self.ensure_model_output_is_correct(model_output)
+
         super(ScorerClassification, self).__init__(model_output)
 
+    def ensure_model_output_is_correct(self, model_output: np.ndarray):
+        if len(model_output) == 0:
+            return
+        if len(model_output.shape) != 2:
+            raise ValueError("ScorerClassification model_output must be a 2-dimensional array")
+        if model_output.shape[1] == 0:
+            raise ValueError("ScorerClassification model_output must not have an empty dimension 1")
+        if model_output.shape[1] == 1:
```

Comment:
while this is an acceptable solution, please consult with @IgorSusmelj whether we want to add this already or only later when the case is not as likely anymore (e.g. when we handle it correctly in the detection scorer). Because right now this can happen relatively often and it's not a good look for us.

Reply:
For now, I just want the tutorials to work o.0
I'm currently running it in background. @philippmwirth or @MalteEbner, could you test the kNN tutorial as well?

Reply:
I am still working on reworking the AL tutorial, I just postponed it every time we changed something on the active learning side (i.e. always).

Reply:
Regarding the warning: I though a little more about it and removed it, as calculating the scores like this makes sense if you have a binary classification problem and thus a vector with the probability of class 0 for each sample (shape: nx1).